### ISSUE TITLE
Repetitive message when I tried to run the code

### ISSUE THREAD

**Issue by torrejoNia** [dhmay/param-medic#7]
Hello! thank you very much for this utility. I have tried to run the code like this:

param-medic \<test2.mzML\>+
>> C:\Users\Me\Documents\Me\DatatobeAnalized\test2\MaxQuantAnalysis

where test2.mzML is the name of the file I want to check. I have also try it in different forms:

param-medic \<test2.mzML\>+ C:\Users\Me\Documents\Me\DatatobeAnalized\test2\MaxQuantAnalysis

param-medic [--options] \<test2.mzML\>+ C:\Users\Me\Documents\Me\DatatobeAnalized\test2\MaxQuantAnalysis

but every time I run the command I get this output text file:

#!C:\Users\Me\Anaconda3\envs\py2\python.exe
# EASY-INSTALL-SCRIPT: 'param-medic==0.2','param-medic'
__requires__ = 'param-medic==0.2'
__import__('pkg_resources').run_script('param-medic==0.2', 'param-medic')

I am new in python, but have consulted with a colleague that has a bit more of experience but he is also not able to find the issue. Would be very grateful if I can have any input.

Thanks in advance.

**Comment by dhmay** [comment]
I'm not able to actively support this tool, any more (and TBH I don't even have it installed on any current machine), but I can point out a couple things.

* The command should be `param-medic test2.mzML`, rather than `param-medic <test2.mzML>+`
    * The "+" in the usage instructions was there to indicate that you could supply more than one file, and the "<>" were to indicate a filename that you supply
* I'm not sure what the filepath starting with "C:" represents. Are you typing that in? You shouldn't be. 
* You should either be running the command from the directory that test2.mzML is in, or supplying the full filepath to test2.mzML in the command

Those changes are necessary, but I'm not sure they'll be sufficient. Have you tried running `pip install param-medic`? That's the easiest way to get up and running.

**Comment by torrejoNia** [comment]
Thank you very much. I have tried using the command  **param-medic test2.mzML** from the directory that test2.mzML is located and I am still obtaining the same .txt file: 

#!C:\Users\Me\Anaconda3\envs\py2\python.exe

EASY-INSTALL-SCRIPT: 'param-medic==0.2','param-medic'
requires = 'param-medic==0.2'
import('pkg_resources').run_script('param-medic==0.2', 'param-medic')

I have installed param-medic already and apparently all is fine, but still cannot run the command. Maybe I am receiving that message because there was a problem during installation?

Thanks in advance, and sorry for asking for support again, if the problem is the installation I will just try to open a ticket on the institution to get it installed properly first.

**Comment by dhmay** [comment]
Yes, what you're encountering isn't anything to do with the param-medic codebase, per se... more of a Python installation issue. I think your institution is probably best set up to help you resolve it. Good luck!
